packages/expo-clipboard/src/Clipboard.types.ts

```diff
@@ -78,3 +80,84 @@ export interface SetStringOptions {
    */
   inputFormat?: StringFormat;
 }
+
+// @needsAudit
+export interface ClipboardPasteButtonProps extends ViewProps {
+  /**
+   * A callback that is called with the result of the paste action.
+   * Inspect the `type` property to determine the type of the pasted data.
+   
+   * Can be one of `text` or `image`.
+   *
+   * @example
+   * ```ts
+   *   onPress={(data) => {
+   *     if (data.type === 'image') {
+   *       setImageData(data);
+   *    } else {
+   *       setTextData(data);
+   *     }
+   *   }}
+   * ```
+   */
+  onPress: (data: PasteEventPayload) => void;
+  /**
+   * The backgroundColor of the button.
+   * Leaving this as the default allows the color to adjust to the system theme settings.
+   *
+   * @default systemBlue
+   * @see [Apple Documentation](https://developer.apple.com/documentation/appkit/nscolor/2879260-systemblue)
```

Comment:
To be honest I think we shouldn't put iOS-specific names to our `@default` in the docs. It kinda implies that I can set this prop to `"systemBlue"` and I will get the default behavior, which is not true because we don't support this named color in our UIColor conversions (afaik).

Reply:
Ok, that makes sense. Should we name the default at all then? I don't think we can name anything that won't break the default behaviour of the system color